Build Failure on ROS 2 Rolling Buildfarm

This is blocking the next package [sync](https://discourse.ros.org/t/preparing-for-rolling-sync-2025-06-27/44474) as `perception` and thus `desktop-full` variants of ROS 2 have regressed.

```
23:13:52 In file included from /tmp/binarydeb/ros-rolling-laser-filters-2.2.0/src/laser_scan_filters.cpp:37:
23:13:52 /tmp/binarydeb/ros-rolling-laser-filters-2.2.0/include/laser_filters/footprint_filter.h:46:10: fatal error: tf2/transform_datatypes.h: No such file or directory
23:13:52    46 | #include <tf2/transform_datatypes.h>
23:13:52       |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
23:13:52 compilation terminated.
```

Detailed logs: https://build.ros2.org/view/Rbin_uN64/job/Rbin_uN64__laser_filters__ubuntu_noble_amd64__binary/107/console

--- a/include/laser_filters/box_filter.h
+++ b/include/laser_filters/box_filter.h
@@ -49,7 +49,7 @@
 
 #include <filters/filter_base.hpp>
 
-#include <tf2/transform_datatypes.h>
+#include <tf2/transform_datatypes.hpp>
 #include <tf2_ros/transform_listener.h>
 #include <sensor_msgs/msg/laser_scan.hpp>
 

--- a/include/laser_filters/footprint_filter.h
+++ b/include/laser_filters/footprint_filter.h
@@ -43,7 +43,7 @@ This is useful for ground plane extraction
 
 #include "filters/filter_base.hpp"
 
-#include <tf2/transform_datatypes.h>
+#include <tf2/transform_datatypes.hpp>
 #include <tf2_ros/buffer.h>
 #include <tf2_ros/transform_listener.h>
 #include <sensor_msgs/msg/laser_scan.hpp>

